src/stock_str.rs

```diff
@@ -443,6 +443,9 @@ pub enum StockMessage {
         fallback = "Could not yet establish guaranteed end-to-end encryption, but you may already send a message."
     ))]
     SecurejoinWaitTimeout = 191,
+
+    #[strum(props(fallback = "Unnamed group"))]
```

Comment:
Btw, i'd suggest to make it "👥📧" or even "..." not to translate it. Not a frequent case anyway

Reply:
Sounds good to me, the less stock strings we need to "carry around" the better

Reply:
Done. I think 👥📧 is the better option